### ISSUE TITLE
Suggestion - Use 'platformdirs' to pick a folder which isn't straight in the home folder

### ISSUE THREAD

**Issue by 59e5aaf4** [CrowdStrike/Falcon-Toolkit#41]
It's a minor suggestion, but I might not be the only one trying to keep the amount of subfolders of ~/ low. I'd love to have Falcon-Toolkit move away from ~/

By the way, the help mentions `~/.FalconToolkit/` but it doesn't exist ?
```
  -c, --config-path TEXT  Path to the configuration directory (default:
                          ~/.FalconToolkit/)
```

The excellent https://github.com/platformdirs/platformdirs project offers a way to automatically put config in a sensible folder like %appdata% or ~/.config/$tool/.

Could it be possible to consider using such a lib to put the default folder away from ~/ ?

Thanks !

**Comment by ChristopherHammond13** [comment]
This is a great suggestion, and I completely agree that having lots of folders under ~ can be irritating (at best!). `platformdirs` looks like a really nice way to do this. I'll need to investigate the best way of handling the migration to ensure it is graceful, but aside from that I do think this is a better way forward.

**Comment by ChristopherHammond13** [comment]
Closed by #53
